pkgs/development/libraries/waylandpp/default.nix

```diff
@@ -14,16 +15,15 @@
 
 assert docSupport -> doxygen != null;
 
```

Comment:
```suggestion
```
Those are no longer required and an artifact from before callPackage. Also please drop ``? null`` from inputs.